app/views/shared/_footer_lite.html.slim

```diff
@@ -4,8 +4,8 @@ footer.footer.bg-navy.white
       .sm-col = t('shared.footer_lite.gsa')
       .sm-col-right.caps
         = link_to t('links.help'), help_path,
-          class: 'white text-decoration-none mr3'
+          class: 'white text-decoration-none mr3', target: 'new'
```

Comment:
should we use `target: '_blank'`? According to this StackOverflow which links to the W3C, `new` is not a real/standard target? http://stackoverflow.com/a/4964223